build.gradle

```diff
@@ -33,13 +33,17 @@ repositories {
 }
 
 android {
-    compileSdkVersion 21
-    buildToolsVersion "21.1.2"
-
+    compileSdkVersion 22
+    buildToolsVersion "22.0.1"
     defaultConfig {
         minSdkVersion 8
         targetSdkVersion 21
     }
+    compileOptions {
+        encoding "UTF-8"
+        sourceCompatibility JavaVersion.VERSION_1_7
```

Comment:
I think we were initially targeting Java version 1.6, but if 1.7 works too then I am fine with this too.